capstone/src/main/resources/static/user.js

```diff
@@ -413,6 +432,8 @@ function getPublicProfile() {
    createProfile(data, false, true);
    const eventDiv = document.getElementById("hosted-events");
    data.events.forEach((event) => eventDiv.appendChild(createSavedEventElement(event, true, false)));
+   document.getElementById("public-image-a").setAttribute("href", "get-public-image?email=" + data.email);
```

Comment:
So is the profile image only supported for organizations?
Or is the comment on this function incorrect?

Reply:
Yes only public organizations will have a public profile for other users to view.